### PR TITLE
Selenium: Create "AccountTest" and related pageobjects

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Create "AccountTest" and related pageobjects

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/8222

### CODE REVIEW

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestTestKeycloakPasswordPagePage.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import org.eclipse.che.selenium.core.SeleniumWebDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+
+@Singleton
+public class TestTestKeycloakPasswordPagePage extends TestKeycloakAbstractPage {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestTestKeycloakPasswordPagePage.java]: Name looks strange. It's better don't start page object class name from "Test" prefix to not to confuse with selenium tests classes.

**Review by tolusha — APPROVED** [review]

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestAccountFields.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.pageobject.dashboard.account;
+
+public class TestAccountFields {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestAccountFields.java]: It's just regular account fields. Nothing special. So, let's name this class as just "Account".

**Review by SkorikSergey — APPROVED** [review]

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestKeycloakAbstractPage.java [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  }
+
+  protected void setFieldValue(By fieldLocator, String value) {
+    waitAndGetElement(fieldLocator).clear();
+    waitFieldValue(fieldLocator, "");
+    waitAndGetElement(fieldLocator).sendKeys(value);
+    waitFieldValue(fieldLocator, value);
+  }
+
+  protected void waitElementIsVisible(By elementLocator) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestKeycloakAbstractPage.java]: This method looks more general than for Keyclock page object.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestKeycloakAbstractPage.java [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+  protected String getFieldValue(By fieldLocator) {
+    return waitAndGetElement(fieldLocator).getAttribute("value");
+  }
+
+  public WebElement waitAndGetElement(By locator) {
+    return loadPageWait.until(visibilityOfElementLocated(locator));
+  }
+
+  private void waitFieldValue(By fieldLocator, String expectedValue) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestKeycloakAbstractPage.java]: This method looks more general than for Keyclock page object.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestKeycloakAbstractPage.java [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    loadPageWait.until(
+        (ExpectedCondition<Boolean>) driver -> getFieldValue(fieldLocator).equals(expectedValue));
+  }
+
+  private void waitAndClickOnElement(By elementLocator) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestKeycloakAbstractPage.java]: This method looks more general than for Keyclock page object.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestKeycloakAbstractPage.java [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+
+  protected void waitElementIsVisible(By elementLocator) {
+    loadPageWait.until(visibilityOfElementLocated(elementLocator));
+  }
+
+  protected String getFieldValue(By fieldLocator) {
+    return waitAndGetElement(fieldLocator).getAttribute("value");
+  }
+
+  public WebElement waitAndGetElement(By locator) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestKeycloakAbstractPage.java]: This method looks more general than for Keyclock page object.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestKeycloakAbstractPage.java [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+    Arrays.asList(
+            By.xpath(ACCOUNT_BUTTON),
+            By.xpath(PASSWORD_BUTTON),
+            By.xpath(AUTHENTICATOR_BUTTON),
+            By.xpath(SESSIONS_BUTTON),
+            By.xpath(APPLICATIONS_BUTTON))
+        .forEach(locator -> waitElementIsVisible(locator));
+  }
+
+  public void switchToNoneParentWindow(String parentWindow) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestKeycloakAbstractPage.java]: This method looks more general than for Keyclock page object.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestKeycloakAbstractPage.java [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+    seleniumWebDriver.switchToNoneCurrentWindow(parentWindow);
+  }
+
+  public void closeWindowAndSwitchToParent(String parentWindow) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestKeycloakAbstractPage.java]: This method looks more general than for Keyclock page object.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestKeycloakAbstractPage.java [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+
+import com.google.inject.Singleton;
+import java.util.Arrays;
+import org.eclipse.che.selenium.core.SeleniumWebDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+@Singleton
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestKeycloakAbstractPage.java]: It doesn't make much sense to set @Singleton in the abstract class

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestKeycloakAbstractPage.java [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+public abstract class TestKeycloakAbstractPage {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestKeycloakAbstractPage.java]: Please, explain the propose of method in the class level documentation

**Review by artaleks9 — APPROVED** [review]

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**